zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/distribution/CommandDistributionBehavior.java

```diff
@@ -252,6 +252,8 @@ public interface RequestBuilder {
     DistributionRequestBuilder unordered();
 
     DistributionRequestBuilder inQueue(String queue);
+
+    ContinuationRequestBuilder afterQueue(String queue);
```

Comment:
❓ Why did you decide to separate the continuation from the disribution itself? Do you think there'll be use-cases where we want to add a continuation without distributing something to a queue? In my mind the api would've been something like:

```java
commandDistributionBehavior
  .withKey(123L)
  .inQueue("queueId")
  .withContinuation(continuationCommand)
  .distribute()
```

Reply:
Yeah exactly. I think it's likely that adding distributions and registering continuations will be two separate steps. One loop is just adding new distributions and something else is registering the continuation. That way we don't have to add logic to the loop to check if it's the last distribution etc.